client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -71,7 +74,14 @@ const StockListComponent: FC = () => {
       Cell: TooltipTextCell,
       width: 350,
     },
-    // TODO:: Add a column for the master list name
+    // TODO: Add back when design has been decided
```

Comment:
Aw I was hoping we could use the `columnDisplayState` but looks like we only support the user hiding columns, rather than having some columns not show by default 🥲